### ISSUE TITLE
Bug prefer-action-creator-in-dispatch

### ISSUE THREAD

**Issue by dangrussell** [timdeschryver/eslint-plugin-ngrx#255]
## Behavior

I'm dispatching an action that takes a props object, which seems to follow the spirit of this rule -- it uses an action creator. Maybe the rule sees the props object in there and thinks it's an action object? It's also possible I could be writing this better :)

## Code snippet

Problem line that trips the rule:
`this.store.dispatch(Auth.Login({ payload: $event }));`

Underlying action imported from another file:
`export const Login = createAction('[Auth] Login', props<{ payload: Authenticate }>());`

## I would be willing to submit a PR to fix this issue :heart:

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[X] No

**Comment by dangrussell** [comment]
Just checked, and this approach does not trigger the rule:

```
const login = { payload: $event };
this.store.dispatch(Auth.Login(login));
```

**Comment by rafaelss95** [comment]
Impressive that you are already using such a new rule 😅! Thanks for the report, I'll take a look ASAP :)

**Comment by tarlepp** [comment]
Yeah, I noticed this same issue on my project - waiting for #256 to be merged and new release of this pluing.
